src/wazuh_modules/wm_azure.c

```diff
@@ -164,8 +172,8 @@ void wm_azure_log_analytics(wm_azure_api_t *log_analytics) {
         switch (wm_exec(command, &output, &status, timeout, NULL)) {
             case 0:
                 if (status > 0) {
-                    mtwarn(WM_AZURE_LOGTAG, "%s: Returned error code: '%d'.", curr_request->tag, status);
-                    mtdebug2(WM_AZURE_LOGTAG, "OUTPUT: %s", output);
+                    mterror(WM_AZURE_LOGTAG, "%s: Returned error code: '%d'.", curr_request->tag, status);
+                    mtdebug1(WM_AZURE_LOGTAG, "OUTPUT: %s", output);
```

Comment:
Why did you change the message from warning to error and the debug level?

Reply:
See response below